docs/demo-plugin.js

```
@@ -0,0 +1,288 @@
+
+    try {
+      // Add styles
+      if (style) {
+        const styleEl = document.createElement('style');
+        styleEl.textContent = style;
+        document.head.appendChild(styleEl);
+      }
+
+      // Create Vue app
+      const scriptContent = script.replace(/export\s+default\s*/, '');
+      const component = eval(`(${scriptContent})`);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)